Traceback (most recent call last):
  File "/Users/mashi/Dropbox/00_works/git/mokumoku-kai/small_works/reliability_test_for_stockdata/av_new.py", line 64, in <module>
    df_av = fetch_from_av(ts, '1min')
            ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/mashi/Dropbox/00_works/git/mokumoku-kai/small_works/reliability_test_for_stockdata/av_new.py", line 42, in fetch_from_av
    data, meta_data = ts.get_intraday(symbol=ticker_symbol,interval=span, outputsize='full')
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/mashi/.venv/lib/python3.11/site-packages/alpha_vantage/alphavantage.py", line 218, in _format_wrapper
    call_response, data_key, meta_data_key = func(
                                             ^^^^^
  File "/Users/mashi/.venv/lib/python3.11/site-packages/alpha_vantage/alphavantage.py", line 160, in _call_wrapper
    return self._handle_api_call(url), data_key, meta_data_key
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/mashi/.venv/lib/python3.11/site-packages/alpha_vantage/alphavantage.py", line 359, in _handle_api_call
    raise ValueError(json_response["Error Message"])
ValueError: Invalid API call. Please retry or visit the documentation (https://www.alphavantage.co/documentation/) for TIME_SERIES_INTRADAY.
Traceback (most recent call last):
  File "/Users/mashi/Dropbox/00_works/git/mokumoku-kai/small_works/reliability_test_for_stockdata/av_new.py", line 64, in <module>
    df_av = fetch_from_av(ts, '1min')
            ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/mashi/Dropbox/00_works/git/mokumoku-kai/small_works/reliability_test_for_stockdata/av_new.py", line 42, in fetch_from_av
    data, meta_data = ts.get_intraday(symbol=ticker_symbol,interval=span, outputsize='full')
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/mashi/.venv/lib/python3.11/site-packages/alpha_vantage/alphavantage.py", line 218, in _format_wrapper
    call_response, data_key, meta_data_key = func(
                                             ^^^^^
  File "/Users/mashi/.venv/lib/python3.11/site-packages/alpha_vantage/alphavantage.py", line 160, in _call_wrapper
    return self._handle_api_call(url), data_key, meta_data_key
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/mashi/.venv/lib/python3.11/site-packages/alpha_vantage/alphavantage.py", line 361, in _handle_api_call
    raise ValueError(json_response["Information"])
ValueError: Thank you for using Alpha Vantage! Our standard API rate limit is 25 requests per day. Please subscribe to any of the premium plans at https://www.alphavantage.co/premium/ to instantly remove all daily rate limits.
Traceback (most recent call last):
  File "/Users/mashi/Dropbox/00_works/git/mokumoku-kai/small_works/reliability_test_for_stockdata/av_new.py", line 64, in <module>
    df_av = fetch_from_av(ts, '1min')
            ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/mashi/Dropbox/00_works/git/mokumoku-kai/small_works/reliability_test_for_stockdata/av_new.py", line 42, in fetch_from_av
    data, meta_data = ts.get_intraday(symbol=ticker_symbol,interval=span, outputsize='full')
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/mashi/.venv/lib/python3.11/site-packages/alpha_vantage/alphavantage.py", line 218, in _format_wrapper
    call_response, data_key, meta_data_key = func(
                                             ^^^^^
  File "/Users/mashi/.venv/lib/python3.11/site-packages/alpha_vantage/alphavantage.py", line 160, in _call_wrapper
    return self._handle_api_call(url), data_key, meta_data_key
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/mashi/.venv/lib/python3.11/site-packages/alpha_vantage/alphavantage.py", line 361, in _handle_api_call
    raise ValueError(json_response["Information"])
ValueError: Thank you for using Alpha Vantage! Our standard API rate limit is 25 requests per day. Please subscribe to any of the premium plans at https://www.alphavantage.co/premium/ to instantly remove all daily rate limits.
Traceback (most recent call last):
  File "/Users/mashi/Dropbox/00_works/git/mokumoku-kai/small_works/reliability_test_for_stockdata/av_new.py", line 64, in <module>
    df_av = fetch_from_av(ts, '1min')
            ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/mashi/Dropbox/00_works/git/mokumoku-kai/small_works/reliability_test_for_stockdata/av_new.py", line 42, in fetch_from_av
    data, meta_data = ts.get_intraday(symbol=ticker_symbol,interval=span, outputsize='full')
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/mashi/.venv/lib/python3.11/site-packages/alpha_vantage/alphavantage.py", line 218, in _format_wrapper
    call_response, data_key, meta_data_key = func(
                                             ^^^^^
  File "/Users/mashi/.venv/lib/python3.11/site-packages/alpha_vantage/alphavantage.py", line 160, in _call_wrapper
    return self._handle_api_call(url), data_key, meta_data_key
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/mashi/.venv/lib/python3.11/site-packages/alpha_vantage/alphavantage.py", line 361, in _handle_api_call
    raise ValueError(json_response["Information"])
ValueError: Thank you for using Alpha Vantage! Our standard API rate limit is 25 requests per day. Please subscribe to any of the premium plans at https://www.alphavantage.co/premium/ to instantly remove all daily rate limits.
Traceback (most recent call last):
  File "/Users/mashi/Dropbox/00_works/git/mokumoku-kai/small_works/reliability_test_for_stockdata/av_new.py", line 64, in <module>
    df_av = fetch_from_av(ts, '1min')
            ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/mashi/Dropbox/00_works/git/mokumoku-kai/small_works/reliability_test_for_stockdata/av_new.py", line 42, in fetch_from_av
    data, meta_data = ts.get_intraday(symbol=ticker_symbol,interval=span, outputsize='full')
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/mashi/.venv/lib/python3.11/site-packages/alpha_vantage/alphavantage.py", line 218, in _format_wrapper
    call_response, data_key, meta_data_key = func(
                                             ^^^^^
  File "/Users/mashi/.venv/lib/python3.11/site-packages/alpha_vantage/alphavantage.py", line 160, in _call_wrapper
    return self._handle_api_call(url), data_key, meta_data_key
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/mashi/.venv/lib/python3.11/site-packages/alpha_vantage/alphavantage.py", line 361, in _handle_api_call
    raise ValueError(json_response["Information"])
ValueError: Thank you for using Alpha Vantage! Our standard API rate limit is 25 requests per day. Please subscribe to any of the premium plans at https://www.alphavantage.co/premium/ to instantly remove all daily rate limits.
Traceback (most recent call last):
  File "/Users/mashi/Dropbox/00_works/git/mokumoku-kai/small_works/reliability_test_for_stockdata/av_new.py", line 64, in <module>
    df_av = fetch_from_av(ts, '1min')
            ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/mashi/Dropbox/00_works/git/mokumoku-kai/small_works/reliability_test_for_stockdata/av_new.py", line 42, in fetch_from_av
    data, meta_data = ts.get_intraday(symbol=ticker_symbol,interval=span, outputsize='full')
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/mashi/.venv/lib/python3.11/site-packages/alpha_vantage/alphavantage.py", line 218, in _format_wrapper
    call_response, data_key, meta_data_key = func(
                                             ^^^^^
  File "/Users/mashi/.venv/lib/python3.11/site-packages/alpha_vantage/alphavantage.py", line 160, in _call_wrapper
    return self._handle_api_call(url), data_key, meta_data_key
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/mashi/.venv/lib/python3.11/site-packages/alpha_vantage/alphavantage.py", line 361, in _handle_api_call
    raise ValueError(json_response["Information"])
ValueError: Thank you for using Alpha Vantage! Our standard API rate limit is 25 requests per day. Please subscribe to any of the premium plans at https://www.alphavantage.co/premium/ to instantly remove all daily rate limits.
Traceback (most recent call last):
  File "/Users/mashi/Dropbox/00_works/git/mokumoku-kai/small_works/reliability_test_for_stockdata/av_new.py", line 64, in <module>
    df_av = fetch_from_av(ts, '1min')
            ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/mashi/Dropbox/00_works/git/mokumoku-kai/small_works/reliability_test_for_stockdata/av_new.py", line 42, in fetch_from_av
    data, meta_data = ts.get_intraday(symbol=ticker_symbol,interval=span, outputsize='full')
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/mashi/.venv/lib/python3.11/site-packages/alpha_vantage/alphavantage.py", line 218, in _format_wrapper
    call_response, data_key, meta_data_key = func(
                                             ^^^^^
  File "/Users/mashi/.venv/lib/python3.11/site-packages/alpha_vantage/alphavantage.py", line 160, in _call_wrapper
    return self._handle_api_call(url), data_key, meta_data_key
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/mashi/.venv/lib/python3.11/site-packages/alpha_vantage/alphavantage.py", line 361, in _handle_api_call
    raise ValueError(json_response["Information"])
ValueError: Thank you for using Alpha Vantage! Our standard API rate limit is 25 requests per day. Please subscribe to any of the premium plans at https://www.alphavantage.co/premium/ to instantly remove all daily rate limits.
Traceback (most recent call last):
  File "/Users/mashi/Dropbox/00_works/git/mokumoku-kai/small_works/reliability_test_for_stockdata/av_new.py", line 64, in <module>
    df_av = fetch_from_av(ts, '1min')
            ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/mashi/Dropbox/00_works/git/mokumoku-kai/small_works/reliability_test_for_stockdata/av_new.py", line 42, in fetch_from_av
    data, meta_data = ts.get_intraday(symbol=ticker_symbol,interval=span, outputsize='full')
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/mashi/.venv/lib/python3.11/site-packages/alpha_vantage/alphavantage.py", line 218, in _format_wrapper
    call_response, data_key, meta_data_key = func(
                                             ^^^^^
  File "/Users/mashi/.venv/lib/python3.11/site-packages/alpha_vantage/alphavantage.py", line 160, in _call_wrapper
    return self._handle_api_call(url), data_key, meta_data_key
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/mashi/.venv/lib/python3.11/site-packages/alpha_vantage/alphavantage.py", line 361, in _handle_api_call
    raise ValueError(json_response["Information"])
ValueError: Thank you for using Alpha Vantage! Our standard API rate limit is 25 requests per day. Please subscribe to any of the premium plans at https://www.alphavantage.co/premium/ to instantly remove all daily rate limits.
Traceback (most recent call last):
  File "/Users/mashi/Dropbox/00_works/git/mokumoku-kai/small_works/reliability_test_for_stockdata/av_new.py", line 64, in <module>
    df_av = fetch_from_av(ts, '1min')
            ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/mashi/Dropbox/00_works/git/mokumoku-kai/small_works/reliability_test_for_stockdata/av_new.py", line 34, in fetch_from_av
    from alpha_vantage.techindicators import TechIndicators
ModuleNotFoundError: No module named 'alpha_vantage'
Traceback (most recent call last):
  File "/Users/mashi/Dropbox/00_works/git/mokumoku-kai/small_works/reliability_test_for_stockdata/av_new.py", line 64, in <module>
    df_av = fetch_from_av(ts, '1min')
            ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/mashi/Dropbox/00_works/git/mokumoku-kai/small_works/reliability_test_for_stockdata/av_new.py", line 34, in fetch_from_av
    from alpha_vantage.techindicators import TechIndicators
ModuleNotFoundError: No module named 'alpha_vantage'
Traceback (most recent call last):
  File "/Users/mashi/Dropbox/00_works/git/mokumoku-kai/small_works/reliability_test_for_stockdata/av_new.py", line 64, in <module>
    df_av = fetch_from_av(ts, '1min')
            ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/mashi/Dropbox/00_works/git/mokumoku-kai/small_works/reliability_test_for_stockdata/av_new.py", line 34, in fetch_from_av
    from alpha_vantage.techindicators import TechIndicators
ModuleNotFoundError: No module named 'alpha_vantage'
Traceback (most recent call last):
  File "/Users/mashi/Dropbox/00_works/git/mokumoku-kai/small_works/reliability_test_for_stockdata/av_new.py", line 64, in <module>
    df_av = fetch_from_av(ts, '1min')
            ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/mashi/Dropbox/00_works/git/mokumoku-kai/small_works/reliability_test_for_stockdata/av_new.py", line 34, in fetch_from_av
    from alpha_vantage.techindicators import TechIndicators
ModuleNotFoundError: No module named 'alpha_vantage'
Traceback (most recent call last):
  File "/Users/mashi/Dropbox/00_works/git/mokumoku-kai/small_works/reliability_test_for_stockdata/av_new.py", line 64, in <module>
    df_av = fetch_from_av(ts, '1min')
            ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/mashi/Dropbox/00_works/git/mokumoku-kai/small_works/reliability_test_for_stockdata/av_new.py", line 34, in fetch_from_av
    from alpha_vantage.techindicators import TechIndicators
ModuleNotFoundError: No module named 'alpha_vantage'
Traceback (most recent call last):
  File "/Users/mashi/Dropbox/00_works/git/mokumoku-kai/small_works/reliability_test_for_stockdata/av_new.py", line 64, in <module>
    df_av = fetch_from_av(ts, '1min')
            ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/mashi/Dropbox/00_works/git/mokumoku-kai/small_works/reliability_test_for_stockdata/av_new.py", line 34, in fetch_from_av
    from alpha_vantage.techindicators import TechIndicators
ModuleNotFoundError: No module named 'alpha_vantage'
Traceback (most recent call last):
  File "/Users/mashi/Dropbox/00_works/git/mokumoku-kai/small_works/reliability_test_for_stockdata/av_new.py", line 64, in <module>
    df_av = fetch_from_av(ts, '1min')
            ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/mashi/Dropbox/00_works/git/mokumoku-kai/small_works/reliability_test_for_stockdata/av_new.py", line 34, in fetch_from_av
    from alpha_vantage.techindicators import TechIndicators
ModuleNotFoundError: No module named 'alpha_vantage'
Traceback (most recent call last):
  File "/Users/mashi/Dropbox/00_works/git/mokumoku-kai/small_works/reliability_test_for_stockdata/av_new.py", line 64, in <module>
    df_av = fetch_from_av(ts, '1min')
            ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/mashi/Dropbox/00_works/git/mokumoku-kai/small_works/reliability_test_for_stockdata/av_new.py", line 34, in fetch_from_av
    from alpha_vantage.techindicators import TechIndicators
ModuleNotFoundError: No module named 'alpha_vantage'
Traceback (most recent call last):
  File "/Users/mashi/Dropbox/00_works/git/mokumoku-kai/small_works/reliability_test_for_stockdata/av_new.py", line 64, in <module>
    df_av = fetch_from_av(ts, '1min')
            ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/mashi/Dropbox/00_works/git/mokumoku-kai/small_works/reliability_test_for_stockdata/av_new.py", line 34, in fetch_from_av
    from alpha_vantage.techindicators import TechIndicators
ModuleNotFoundError: No module named 'alpha_vantage'
Traceback (most recent call last):
  File "/Users/mashi/Dropbox/00_works/git/mokumoku-kai/small_works/reliability_test_for_stockdata/av_new.py", line 64, in <module>
    df_av = fetch_from_av(ts, '1min')
            ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/mashi/Dropbox/00_works/git/mokumoku-kai/small_works/reliability_test_for_stockdata/av_new.py", line 34, in fetch_from_av
    from alpha_vantage.techindicators import TechIndicators
ModuleNotFoundError: No module named 'alpha_vantage'
Traceback (most recent call last):
  File "/Users/mashi/Dropbox/00_works/git/mokumoku-kai/small_works/reliability_test_for_stockdata/av_new.py", line 64, in <module>
    df_av = fetch_from_av(ts, '1min')
            ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/mashi/Dropbox/00_works/git/mokumoku-kai/small_works/reliability_test_for_stockdata/av_new.py", line 34, in fetch_from_av
    from alpha_vantage.techindicators import TechIndicators
ModuleNotFoundError: No module named 'alpha_vantage'
Traceback (most recent call last):
  File "/Users/mashi/Dropbox/00_works/git/mokumoku-kai/small_works/reliability_test_for_stockdata/av_new.py", line 64, in <module>
    df_av = fetch_from_av(ts, '1min')
            ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/mashi/Dropbox/00_works/git/mokumoku-kai/small_works/reliability_test_for_stockdata/av_new.py", line 34, in fetch_from_av
    from alpha_vantage.techindicators import TechIndicators
ModuleNotFoundError: No module named 'alpha_vantage'
Traceback (most recent call last):
  File "/Users/mashi/Dropbox/00_works/git/mokumoku-kai/small_works/reliability_test_for_stockdata/av_new.py", line 64, in <module>
    df_av = fetch_from_av(ts, '1min')
            ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/mashi/Dropbox/00_works/git/mokumoku-kai/small_works/reliability_test_for_stockdata/av_new.py", line 34, in fetch_from_av
    from alpha_vantage.techindicators import TechIndicators
ModuleNotFoundError: No module named 'alpha_vantage'
